functional-tests/CloudbaseInit.Tests.ps1

```diff
@@ -287,6 +301,29 @@ function after.cloudbaseinit.plugins.windows.bootconfig.BootStatusPolicyPlugin {
 
 function prepare.nocloud {
     pushd "$here/../$($env:CLOUD)"
+
+        $openVpnUrl = "https://build.openvpn.net/downloads/releases/OpenVPN-2.5.10-I601-amd64.msi"

```

Comment:
Do we need OpenVPN or can we just install the tap driver from here? https://github.com/OpenVPN/tap-windows6/releases

We should add an inline comment, saying why we need OpenVPN.

Reply:
We need OpenVPN. Installing the driver only does not do anything.

Reply:
We can simply unzip the archive and then manually create the tap device like so:

```
PS E:\tools\tap> .\devcon.exe install .\OemVista.inf tap0901
Device node created. Install is complete when drivers are installed...
Updating drivers for tap0901 from E:\tools\tap\OemVista.inf.
Drivers installed successfully.

PS E:\tools\tap> get-netadapter -InterfaceDescription "*TAP*"

Name                      InterfaceDescription                    ifIndex Status       MacAddress             LinkSpeed
----                      --------------------                    ------- ------       ----------             ---------
Local Area Connection     TAP-Windows Adapter V9                       25 Disconnected 00-FF-92-31-A1-57         1 Gbps
```

Reply:
I suggest to leave the MSI installation, as it is only 4MB and requires one liner (simplicity ++). Also, if the internal ZIP contents change, a fix would be required. Hopefully, the MSI installation will be good for a long time.